### PR TITLE
NEWS: News update for RC6

### PR DESCRIPTION
**Pull request by shamisp**
News update.

### CODE REVIEW

**Comment by shamisp** [comment]
ok to test

**Comment by shamisp** [comment]
Ok to test

File: NEWS [hunk 1]
```diff
@@ -56,6 +57,8 @@
 - Fixes in memcpy selection for AMD
 - Fixed in UCT/UD endpoint flush functionality
 - Fixes in XPMEM detection
+- Fixes in rendezvous staging protocol
+- Fixes in ROCE mlx5 UD port configuration
```
Comment (yosefe) [inline NEWS:61]: Fixes in RoCEv1 UDP.src_port configuration with DevX

File: NEWS [hunk 2]
```diff
@@ -7,7 +7,7 @@
 ##
 #
 
-## 1.9.0-rc5 (September 10, 2020)
+## 1.9.0-rc6 (September 14, 2020)
```
Comment (yosefe) [inline NEWS:10]: Sep 15

File: NEWS (hunk 1 above)
Comment (shamisp) [reply NEWS:61]: is this UDP or UD ?  UDP.src_port  - too much details.

Comment (yosefe) [reply NEWS:61]: it's UDP (layer4). RDMA headers (UD/RC/..) and after layer4 headers.

Comment (shamisp) [reply NEWS:61]: kk

**Comment by shamisp** [comment]
@yosefe please take a look. I don't want to wait for a full ci.

**Review by yosefe — APPROVED** [review]

**Pull request merged**